src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -6946,7 +6950,7 @@ GetDacTableAddress(ICorDebugDataTarget* dataTarget, ULONG64 baseAddress, PULONG6
         return E_INVALIDARG;
     }
 #endif
-    // On MacOS, FreeBSD or NetBSD use the RVA include file
+    // On FreeBSD, NetBSD, or SunOS use the RVA include file
     *dacTableAddress = baseAddress + DAC_TABLE_RVA;
```

Comment:
Unless this is just a placeholder for these platform, this isn't going to work. DAC_TABLE_RVA is the offset to the g_dacTable.

Reply:
Never mind. I looked at the diff incorrectly.  I thought this code was in your new code in ClrDataAccess::Initialize() where you are getting the DotNetRuntimeContractDescriptor export address.